### ISSUE TITLE
Cannot locate an issue by typing the issue number in filter

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#1121]
e.g. The filter `#nnnn` should have the issue #nnnn among results, right?

**Comment by acruis** [comment]
Isn't this `id:nnnn`?

**Comment by damithc** [comment]
Yes, that's the filter. I'm thinking if a user typed the id as a normal search string, we should still be able to find the issue? What do you think?

**Comment by dariusf** [comment]
We could make it so `#n` is parsed as `id:n`? Or maybe as `(id:n || "#n")`.

**Comment by damithc** [comment]
Latter seems safer
